example/src/DrawerExample.re

```diff
@@ -1,27 +1,21 @@
 open BsReactNavigation;
-open Config;
 
 module Drawer =
   DrawerNavigation.Create({
+    open Config;
+    open DrawerNavigation;
+
     type item = Config.item;
 
     let items = [Dashbord, Settings];
-    let drawerOptions =
-      DrawerNavigation.drawerOptions(~activeTintColor="#847", ());
-
-    let order = [Dashbord, Settings];
+    let drawerOptions = drawerOptions(~drawerWidth=200, ());
```

Comment:
We are inside `drawer` already, so `drawer` in `drawerWidth` is not necessary. Can be just `width`

Reply:
It's taken from [DOC](https://reactnavigation.org/docs/en/drawer-navigator.html). I'd like to keep naming form `ract-navigation`